### DISCUSSION TITLE
initial condition error in liionpack

### DISCUSSION THREAD

**Discussion by balaji3112** [pybamm-team/PyBaMM#2753] [Q&A]
While using my own drive cycle file as an input I am getting the following error:  
pybamm.expression_tree.exceptions.SolverError: Events ['Maximum voltage'] are non-positive at initial conditions

Is there something wrong with my initial conditions?
netlist = lp.setup_circuit(Np=1, Ns=1, Rb=1e-4, Rc=1e-3, Ri=5e-2, V=4.3)

I'm attaching the csv file of my input data below.
[pulse_discharge.csv](https://github.com/pybamm-team/PyBaMM/files/10917729/pulse_discharge.csv)

**Comment by TomTranter** [comment]
Should be ok, are you using version 0.3.3?

**Reply by balaji3112** [reply]
Yeah, I am using the same version.

**Comment by balaji3112** [comment]
The issue has been resolved. there was a problem in passing the drive cycle file through the experiment class now it has been resolved.
